app/code/Magento/CheckoutAgreements/Block/Adminhtml/Agreement/Edit/Form.php

```diff
@@ -59,6 +81,8 @@ protected function _construct()
     protected function _prepareForm()
     {
         $model = $this->_coreRegistry->registry('checkout_agreement');
+        $wysiwygConfig = $this->_wysiwygConfig->getConfig();
+        $showEditor = $this->scopeConfig->getValue(self::XML_PATH_SHOW_EDITOR);
```

Comment:
I don't think these variables are necessary as they are used only once, feel free to perform a call directly in the place when the return is needed